components/common/dialogComponent/dialogComponent.tsx

```diff
@@ -0,0 +1,123 @@
+import React from "react";
+import { Notification, Toaster } from "rsuite";
+
+interface DialogComponentProps {
+  type: "error" | "info" | "success" | "warning";
+  header: string;
+  body?: string;
+}
+/**
+ *
+ * @param prop - type (error, info, success ), header - title text, body - body text
```

Comment:
Let's add the descriptions for each prop, above each prop as a comment inside the `DialogComponentProps` itself.

Ref: https://github.com/wso2/identity-apps/blob/master/modules/react-components/src/components/avatar/animated-avatar.tsx#L26-L38

```suggestion
 * @param prop - props for the component
```